### PR TITLE
Conda recipe for RISE

### PR DESCRIPTION
**Pull request by ijstokes**
Just tested on my OS X 10.10 laptop using latest Jupyter conda package as base.  But seems to work.

### CODE REVIEW

**Comment by ijstokes** [comment]
The `meta.yaml` file specifies my forked repo GH URL, instead of the actual project repo.

File: conda.recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    number: 1
+
+source:
+  git_url: https://github.com/ijstokes/RISE.git
+  git_tag: master
+
+requirements:
+  build:
+    - python
+    - jupyter
```
Comment (damianavila) [inline conda.recipe/meta.yaml]: jupyter is the metackage installing all... there is no need to add it here... only the notebook should be necessary

File: conda.recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package:
+  name: rise
+  version: 0.3.0
+
+build:
+    number: 1
+
+source:
+  git_url: https://github.com/ijstokes/RISE.git
```
Comment (damianavila) [inline conda.recipe/meta.yaml]: This has to point to the main repo... as you mnetioned in some comment...

File: conda.recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package:
+  name: rise
+  version: 0.3.0
```
Comment (damianavila) [inline conda.recipe/meta.yaml]: This should be `3.x`... in fact, we probably need to add a `version` file somewhere in the future, but that's for later...

File: conda.recipe/build.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+SRC_DIR=$RECIPE_DIR/..
+cd $SRC_DIR
```
Comment (damianavila) [inline conda.recipe/build.sh]: you are missing npm-related steps here... check: https://github.com/damianavila/RISE#development (the same goes for the `bld.bat` file.

**Comment by damianavila** [comment]
@ijstokes I left some comments here...

**Comment by damianavila** [comment]
@ijstokes closing in favor of #204
Thanks for your contribution!

**Pull request closed without merging**